### PR TITLE
Story 10: Add new alternative

### PR DESCRIPTION
**Pull request by kimc0de**
- [x] input fields
- [x] drop image (js)
- [x] responsive

### CODE REVIEW

**Comment by DariaBezuhla** [comment]
closing it

File: public/stylesheets/newApp.css [hunk 1]
```diff
@@ -1,23 +1,62 @@
...
+  color: var(--text-primary);
+  width: 100%;
+}
+
+#dropdownCategoriesButton {
+  width: 65%;
+  background-color: var(--button-primary);
+  border-radius: 20px;
+  border: none;
+}
```
Comment (kimc0de) [inline public/stylesheets/newApp.css:23]:
```suggestion
#dropdownCategoriesButton {
  width: 65%;
  background-color: var(--button-primary);
  border-radius: 20px;
  border: none;
}
```

File: public/stylesheets/newApp.css [hunk 2]
```diff
@@ -1,23 +1,62 @@
...
+  padding-bottom: 15px;
+}
+
+.dropdown {
+  position: relative;
+}
+
+.dropdown-menu {
+  width: 63%;
+}
```
Comment (kimc0de) [inline public/stylesheets/newApp.css:11]: #new-app .dropdown-menu {
    width: 63%;
    will-change: transform;
    left: 50%;
    right: auto;
    text-align: center;
    transform: translate(-50%, 0);
}
```suggestion
.dropdown-menu {
  width: 63%;
}
```

File: public/stylesheets/newApp.css [hunk 3]
```diff
@@ -1,23 +1,62 @@
+#new-app {
+  padding-bottom: 15px;
+}
+
+.dropdown {
```
Comment (kimc0de) [inline public/stylesheets/newApp.css:5]:
```suggestion
#new-app .dropdown {
```

File: public/stylesheets/newApp.css [hunk 4]
```diff
@@ -1,23 +1,62 @@
...
+}
+
+.dropdown-menu {
+  width: 63%;
+}
+
+.dropdown-item {
+  color: var(--text-primary);
+  width: 100%;
+}
```
Comment (kimc0de) [inline public/stylesheets/newApp.css:16]:
```suggestion
#new-app .dropdown-item {
  color: var(--text-primary);
  width: 100%;
}
```

File: public/stylesheets/newApp.css [hunk 5]
```diff
@@ -1,23 +1,67 @@
...
+
+#dropdownCategoriesButton {
+  width: 65%;
+  background-color: var(--button-primary);
+  border-radius: 20px;
+  border: none;
+}
```
Comment (kimc0de) [inline public/stylesheets/newApp.css:28]:
```suggestion
#dropdownCategoriesButton {
    width: 100%;
    background-color: var(--input-primary);
    border-radius: 10px;
    border: none;
    color: var(--text-primary);
}
```

File: public/stylesheets/newApp.css [hunk 6]
```diff
@@ -1,23 +1,68 @@
...
+
+#new-app .dropdown-menu {
+    width: 63%;
+    will-change: transform;
+    left: 50%;
+    right: auto;
+    text-align: center;
+    transform: translate(-50%, 0);
+}
```
Comment (kimc0de) [inline public/stylesheets/newApp.css:16]:
```suggestion
#new-app .dropdown-menu {
    width: 63%;
}

#new-app .dropdown-menu.show {
  will-change: transform;
  top: auto !important;
  left: 50% !important;
  right: auto;
  text-align: center;
  transform: translate(-50%, 0) !important;
}
```

**Pull request merged**